src/docs.js

```diff
@@ -1,4 +1,35 @@
-function documenter() {
+let tar = require('tar-stream');
+
+async function documenter(options = {}) {
+  // print schemas
+  console.log(options.schemas);
+
+  // take schemas(name + content) + add it to stream tarball
+  // pack is a streams2 stream
+  let tarball = tar.pack();
+
+  let schemas = options.schemas;
+
+  if (schemas) {
```

Comment:
Note: if we default `schemas` to be `[]` we don't need this extra check... That's nice :)